sys/riscv/riscv/pmap.c

```diff
@@ -935,8 +935,13 @@ pmap_bootstrap(vm_paddr_t kernstart, vm_size_t kernlen)
 	 */
 	freeva = freemempos - kernstart + KERNBASE;
 #ifdef __CHERI_PURE_CAPABILITY__
-	freeva = (vm_pointer_t)cheri_setbounds(cheri_setaddress(
-	    kernel_root_cap, freeva), VM_MAX_KERNEL_ADDRESS - freeva);
+	freeva = (vm_pointer_t)cheri_setaddress(
```

Comment:
This indentation is not the typical style used in FreeBSD.  In this case I think it is more readable to use a temporary to define the bounds, something like:

```
#ifdef __CHERI_PURE_CAPABILITY__
       void *kmemcap;
#endif
....
#ifdef __CHERI_PURE_CAPABILITY
        /* Comment explaining why these bounds are needed. */
        kmemcap = cheri_setaddress(kernel_root_cap, VM_MIN_KERNEL_ADDRESS);
        kmemcap = cheri_setbounds(kmemcap, VM_MAX_KERNEL_ADDRESS - VM_MIN_KERNEL_ADDRESS);
        freeva = (vm_pointer_t)cheri_setaddress(kmemcap, freeva);
#endif
```

Reply:
Done - I merged the #ifdef for the kmemcap = ... with the one for the #define reserve_space, because they both interact with the 'freeva' variable in a CHERI-aware way that matters